packages/nbdime/src/styles/variables.css

```diff
@@ -27,4 +27,10 @@
 
   --jp-merge-either-color1: #aee;
   --jp-merge-either-color2: #cff4f4;
-}
\ No newline at end of file
+  --jp-expand-outer-wrapper: rgb(221, 244, 255);
+  --jp-expand-color: rgb(87, 96, 106);
+  --jp-expand-a-color: rgba(84, 174, 255, 0.4)
```

Comment:
Nitpick: Ideally concepts of the DOM should not bleed into the CSS variable names. Ideally it would say something about the intended difference/relationship between the different variables, so that it can more easily by themed by others (and have those themes survive a change to the DOM).

Also, this color has an alpha element in order to work on multiple themed backgrounds. The other colors in these variables should ideally also work in the same constraints. Mainly since default JupyterLab themes do not include nbdime theming, but we still want nbdime to work with all of them.

Reply:
Ok I changed these to not reflect the dom elements that use them.